src/Modules/PdfModule.ts

```diff
@@ -1,5 +1,5 @@
-import { getPdfData} from '../Pdf';
-import { PdfData, PdfQuery } from '../Pdf/PdfData';
+import { generatePdf, getPdfData } from '../Pdf';
+import { GeneratingState, PdfData, PdfQuery } from '../Pdf/PdfData';
```

Comment:
I know it's my fault, but let's import everything from `'../Pdf/PdfData'` and reexport it in `'../Pdf'` ... so we import everything from single namespace.